### PR TITLE
Antivirus registration enabled by default

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves https://github.com/elastic/security-docs/issues/5472 — updates both classic & serverless docs.

### Previews

Classic:
   * [Configure an integration policy for Elastic Defend | Register Elastic Security as antivirus (optional)](https://security-docs_bk_5494.docs-preview.app.elstc.co/guide/en/security/master/configure-endpoint-integration-policy.html#register-as-antivirus)

Serverless:
   * Click the [link below](https://github.com/elastic/security-docs/pull/5494#issuecomment-2204119916), then in the Security card click **View serverless docs**.
   * Navigate to: **Secure your endpoints → Configure an integration policy | Register Elastic Security as antivirus (optional)**

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-8trnwe13y-elastic-dev.vercel.app

* Built with commit 35c461f7c35eb46ae803fd66aa7403de8f08c293

> Issues? Visit #next-docs in Slack

**Review by szwarckonrad — APPROVED** [review]
LGTM!

**Review by ferullo — APPROVED** [review]

File: docs/serverless/edr-install-config/configure-endpoint-integration-policy.mdx [hunk 1]
```diff
@@ -249,7 +249,9 @@ register ((elastic-sec)) as your hosts' antivirus software by enabling **Registe
...
 </DocCallOut>
 
-You can also choose **Sync with malware protection level** to automatically set antivirus registration based on how you've configured ((elastic-defend))'s <DocLink slug="/serverless/security/configure-endpoint-integration-policy" section="malware-protection">malware protection</DocLink>. If malware protection is turned on and set to **Prevent**, antivirus registration will also be enabled; in any other case, antivirus registration will be disabled.
+By default, the **Sync with malware protection level** is selected to automatically set antivirus registration to match how you've configured ((elastic-defend))'s <DocLink slug="/serverless/security/configure-endpoint-integration-policy" section="malware-protection">malware protection</DocLink>. If malware protection is turned on _and_ set to **Prevent**, antivirus registration will also be enabled; in any other case, antivirus registration will be disabled.
+
+If you don't want to sync antivirus registration, you can set it manually with the **Enabled** or **Disabled** option.
```
Comment (nastasha-solomon) [inline docs/serverless/edr-install-config/configure-endpoint-integration-policy.mdx:254]:
```suggestion
If you don't want to sync antivirus registration, you can set it manually with the **Enabled** or **Disabled** options.
```

**Review by nastasha-solomon — APPROVED** [review]
Just one potential nit - lgmt, otherwise!

File: docs/getting-started/configure-integration-policy.asciidoc [hunk 2]
```diff
@@ -212,7 +212,9 @@ register {elastic-sec} as your hosts' antivirus software by enabling **Register
 
 NOTE: Windows Server versions are not supported. Antivirus registration requires Windows Security Center, which is not included in Windows Server operating systems.
 
-You can also choose **Sync with malware protection level** to automatically set antivirus registration based on how you've configured {elastic-defend}'s <<malware-protection,malware protection>>. If malware protection is turned on and set to **Prevent**, antivirus registration will also be enabled; in any other case, antivirus registration will be disabled.
+By default, the **Sync with malware protection level** is selected to automatically set antivirus registration to match how you've configured {elastic-defend}'s <<malware-protection,malware protection>>. If malware protection is turned on _and_ set to **Prevent**, antivirus registration will also be enabled; in any other case, antivirus registration will be disabled.
+
+If you don't want to sync antivirus registration, you can set it manually with the **Enabled** or **Disabled** option.
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/getting-started/configure-integration-policy.asciidoc:217]: I know that for subject-verb agreement, it depends whether the compound subjects are joined by `and` or `or`. But not sure if that applies to... uh... compound adjectives? Regardless we could just simplify:
```suggestion
If you don't want to sync antivirus registration, you can set it manually with **Enabled** or **Disabled**.
```

File: docs/serverless/edr-install-config/configure-endpoint-integration-policy.mdx (hunk 1 above)
Comment (joepeeples) [reply docs/serverless/edr-install-config/configure-endpoint-integration-policy.mdx:254]:
```suggestion
If you don't want to sync antivirus registration, you can set it manually with **Enabled** or **Disabled**.
```

**Pull request merged**